BuildResidentialScheduleFile/measure.rb

```diff
@@ -97,41 +102,76 @@ def run(model, runner, user_arguments)
     end
     args[:hpxml_output_path] = hpxml_output_path
 
-    hpxml = HPXML.new(hpxml_path: hpxml_path)
+    hpxml = HPXML.new(hpxml_path: hpxml_path, building_id: 'ALL')
 
-    # FIXME: Relax this constraint (using a new building_id measure argument?)
-    if hpxml.buildings.size > 1
-      runner.registerError('Cannot currently handle an HPXML with multiple Building elements.')
-      return false
+    debug = false
+    if args[:debug].is_initialized
+      debug = args[:debug].get
     end
-    hpxml_bldg = hpxml.buildings[0]
+    args[:debug] = debug
 
-    # exit if number of occupants is zero
-    if hpxml_bldg.building_occupancy.number_of_residents == 0
-      runner.registerInfo('Number of occupants set to zero; skipping generation of stochastic schedules.')
-      return true
+    if hpxml.buildings.size > 1 && !args[:building_id].is_initialized
+      fail "Argument 'building_id' required if there are multiple Building elements in the HPXML file."
     end
 
-    # create EpwFile object
-    epw_path = Location.get_epw_path(hpxml_bldg, hpxml_path)
-    epw_file = OpenStudio::EpwFile.new(epw_path)
+    # random seed
```

Comment:
Move the random seed stuff from the schedules class to here, so that we can adjust its value based on the unit of the building.